librosa/core/notation.py

```diff
@@ -122,16 +123,17 @@
 
 
 # Pre-compiled regular expressions for note and key parsing
+KEY_RE = re.compile(
+    r"^(?P<tonic>[A-Ga-g])" r"(?P<accidental>([#♯b!♭𝄪𝄫]*))" r":(?P<scale>(maj|min)(or)?)$"
+)
```

Comment:
At this point, the first parts of KEY_RE and NOTE_RE are (rightfully) nearly identical, except that KEY doesn't support naturals.  Would it make sense to include that here too?